tests/unit/amo/components/TestVPNPromoBanner.js

```diff
@@ -50,7 +51,8 @@ describe(__filename, () => {
     dispatchClientMetadata({ store, clientApp, regionCode });
     return shallowUntilTarget(
       <VPNPromoBanner
-        _tracking={_tracking}
+        _tracking={createFakeTracking()}
+        _window={{ localStorage: createFakeLocalStorage() }}
```

Comment:
Seeing this, I am thinking that it might be wiser to only inject `localStorage` as opposed to the full `window` object.

Reply:
Good idea!